helm/hpcc/templates/_helpers.tpl

```diff
@@ -68,19 +68,49 @@ Translate a port list to a comma-separated list
  {{- end -}}
 {{- end -}}
 
+{{- define "hpcc.getFirstPlaneForLabel" -}}
+{{- $root := .root -}}
```

Comment:
style: Not sure what you gain by this rather than just using .root in the code below?